src/player-statistics.js

```diff
@@ -83,9 +84,9 @@ export default [
   {group: 'Raids', title: 'Gaeting Crystals', key: 'statistics.gaetingCrystals', sprite: 'gaeting-crystal', gold: false, by_playtime: true, decimals: 0},
   {group: 'Progression', title: 'Finished Achievements', key: 'statistics.achievementCount', sprite: 'account-achievements', gold: false, by_playtime: true, decimals: 0},
   {group: 'Progression', title: 'Unlocked Skins (Total)', key: 'statistics.skinCount', sprite: 'account-wardrobe', gold: false, by_playtime: true, decimals: 0},
-  // {group: 'Progression', title: 'Unlocked Skins (Weapons)', key: 'statistics.weaponSkinCount', sprite: 'account-wardrobe', gold: false, by_playtime: true, decimals: 0},
-  // {group: 'Progression', title: 'Unlocked Skins (Armor)', key: 'statistics.armorSkinCount', sprite: 'account-wardrobe', gold: false, by_playtime: true, decimals: 0},
-  // {group: 'Progression', title: 'Unlocked Skins (Backpieces)', key: 'statistics.backSkinCount', sprite: 'account-wardrobe', gold: false, by_playtime: true, decimals: 0},
+  {group: 'Progression', title: 'Unlocked Skins (Weapons)', key: 'statistics.weaponSkinCount', sprite: 'account-wardrobe', gold: false, by_playtime: true, decimals: 0},
+  {group: 'Progression', title: 'Unlocked Skins (Armor)', key: 'statistics.armorSkinCount', sprite: 'account-wardrobe', gold: false, by_playtime: true, decimals: 0},
+  {group: 'Progression', title: 'Unlocked Skins (Backpieces)', key: 'statistics.backSkinCount', sprite: 'account-wardrobe', gold: false, by_playtime: true, decimals: 0},
```

Comment:
Can you delete these from the sheet and re-run the generator? We're not having them anymore.